depthmapXcli/runmethods.cpp

```diff
@@ -379,4 +379,22 @@ namespace dm_runmethods
         std::cout << " ok" << std::endl;
     }
 
+    void exportData(const CommandLineParser &cmdP, const ExportParser &exportP, IPerformanceSink &perfWriter ) {
+
+        std::unique_ptr<Communicator> comm(new ICommunicator());
+
+        auto mgraph = loadGraph(cmdP.getFileName().c_str(), perfWriter);
+
+        PointMap& currentMap = mgraph->getDisplayedPointMap();
+
+        switch(exportP.getExportMode()) {
```

Comment:
This switch statement should probably at least print a help message as default if not throw.